### ISSUE TITLE
Harmonize python tool configuration

### ISSUE THREAD

**Issue by tmadlener** [AIDASoft/podio#703]
Currently we have a `.flake8`, a `setup.cfg` and a `pyproject.toml` with partially contradictory settings. These should all be in one place only and consistent.

**Comment by m-fila** [comment]
How about going with ruff?

**Comment by tmadlener** [comment]
Yeah probably a good idea to simply switch everything over to that to be consistent with the rest of Key4hep.
